src/pngtocss.c

```diff
@@ -19,11 +19,8 @@
 
 typedef enum {
 	OK = 0,
-	E_NOT_PNG,
```

Comment:
Were these removed intentionally or accidentally?  I think they need to be here.